change/@internal-calling-stateful-client-eb705db4-7ad2-4024-a1e6-0d49a85188b2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add TeamsCall and TeamsCallAgent to stateful - Add new beta feature flag \"teams-call\" - Extract common logic CallAgentCommon and CallCommon",
```

Comment:
nit: these comments are for the public changelogs, `Add new beta feature flag "teams-call"` doesn't apply for external developers of a changelog

Reply:
nit: `stateful` -> `StatefulCallClient`